### PR TITLE
Peter's changes

### PR DESCRIPTION
**Pull request by inglesp**
Addresses #42.

Without being able to run the code, I've got low-medium confidence that this will work, but it's a start...

### CODE REVIEW

File: notebooks/utilities2.py [hunk 1]
```diff
@@ -30,6 +30,17 @@ def suppress_and_round(df, field="row_count", keep=False):
     return df, suppressed
 
 
+def round_and_suppress(df, field):
+    """Another function to apply disclosure control to a column in a dataframe.
+
+    This one rounds all values to the nearest 5, and then replaces any values less than
+    or equal to 5 with the string "1-7".
```
Comment (LindaNab) [inline notebooks/utilities2.py:37]: this should be "any value less or equal to 7 with the string "1-7""

File: notebooks/utilities2.py [hunk 2]
```diff
@@ -30,6 +30,17 @@ def suppress_and_round(df, field="row_count", keep=False):
...
+    """
+    df[field] = (5 * (df[field] / 5).round()).astype(int)
+    df.loc[df[field] <= 5, "n"] = "1-7"
```
Comment (LindaNab) [inline notebooks/utilities2.py:40]: <=7 not <=5

**Review by LindaNab — CHANGES_REQUESTED** [review]
Looks great (as far as I can judge :-)). Two small comments about suppressing <=7 and not <=5.

File: notebooks/utilities2.py (hunk 1 above)
Comment (inglesp) [reply notebooks/utilities2.py]: Values between 5 and 7 will already have been rounded to 5, so this is correct.  But I've updated the comment slightly for clarity.

Comment (LindaNab) [reply notebooks/utilities2.py]: Ah, yes of course! Thanks :-)

**Pull request merged**